bpf/Makefile

```diff
@@ -40,4 +40,6 @@ install:
 	cp -r include/* $(DESTDIR)$(LIBDIR)/cilium/include/
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(LIBDIR)/cilium/probes/
 	cp -r probes/* $(DESTDIR)$(LIBDIR)/cilium/probes/
+	find $(DESTDIR)$(LIBDIR)/cilium -type d -exec chmod 755 {} \;
+	find $(DESTDIR)$(LIBDIR)/cilium -type f -exec chmod 644 {} \;
```

Comment:
Why is this needed? Are the permissions in the git tree wrong? There are scripts in $(LIBDIR)/cilium which have the execute bit set which would be overwritten by this.